### PR TITLE
Add silent flag

### PR DESCRIPTION
**Pull request by waybackarchiver**

### CODE REVIEW

File: downloader/downloader.go [hunk 1]
```diff
@@ -49,11 +50,15 @@ type Downloader struct {
...
+func (downloader *Downloader) progressBar(size int64) *pb.ProgressBar {
 	tmpl := `{{counters .}} {{bar . "[" "=" ">" "-" "]"}} {{speed .}} {{percent . | green}} {{rtime .}}`
+	width := 1000
+	if downloader.option.Silent {
+		tmpl, width = ``, 0
+	}
 	return pb.New64(size).
 		Set(pb.Bytes, true).
-		SetMaxWidth(1000).
+		SetMaxWidth(width).
```
Comment (iawia002) [inline downloader/downloader.go:61]: I think we should not start the progress bar instead of hiding the output

**Comment by iawia002** [comment]
And could you please update the README documentation?

**Comment by waybackarchiver** [comment]
done

**Comment by iawia002** [comment]
LGTM, thanks!

**Pull request merged**